packages/block-library/src/navigation/edit/inner-blocks.js

```diff
@@ -98,7 +104,7 @@ export default function NavigationInnerBlocks( {
 			onChange,
 			allowedBlocks: ALLOWED_BLOCKS,
 			prioritizedInserterBlocks: PRIORITIZED_INSERTER_BLOCKS,
-			__experimentalDefaultBlock: DEFAULT_BLOCK,
+			__experimentalDefaultBlock: navDefaultBlock,
```

Comment:
Given this was introduced as an experimental API, I'm not sure about exposing it as a stable attribute of a block.

It might be worth stabilizing the `__experimentalDefaultBlock` API first so that this becomes `defaultBlock: navDefaultBlock` and deprecating the `__experimentalDefaultBlock` property.

Reply:
Resolved by https://github.com/WordPress/gutenberg/pull/52083